scripts/test_tutorials.py

```diff
@@ -39,6 +38,7 @@ def test_notebook(self, path):
                 src = cell['source']
                 # Comment out lines containing '!' but not '!='
                 src = re.sub(r'\!(?!=)', r'#!', src)
+                src = re.sub('NUM_EXAMPLES ?= ?.*', 'NUM_EXAMPLES = 10', src)

```

Comment:
Can you make a note that this is for mnist notebook ?

Reply:
done.